plugins/vpc-eni/config/netconfig.go

```diff
@@ -44,6 +45,7 @@ type netConfigJSON struct {
 	ENIIPAddresses     []string `json:"eniIPAddresses"`
 	GatewayIPAddresses []string `json:"gatewayIPAddresses"`
 	UseExistingNetwork bool     `json:"useExistingNetwork"`
+	OpState            bool     `json:"opState"`
```

Comment:
Is this the desired operational state of the ENI in the container netns? What is the usecase when the desired operational state is down?

Reply:
OpState down is used in some multi-eni scenarios where we want the second ENI to remain down because it is configured by a separate process after the CNI plugin exits.

Reply:
Also I think we need to use this field to set `netConfig.OpState` on L81.

Reply:
Thanks for the explanation!